woocommerce/woocommerce-square
# Add support for the product blocks.

### All Submissions:



---

### Changes proposed in this Pull Request:
The PR adds support for the product blocks of WooCommerce (e.g., Product Collection block, Top-rated Products block, etc.). Square adds a gift card product, which comes with some fields to fill out before adding it to the cart, and also has custom add-to-cart text. This PR ensures that these features work with product blocks as well.

Closes #120

### Steps to test the changes in this Pull Request:
1. Test with TT4 (Twenty Twenty-Four) and Storefront Theme
1. Check and verify that the Giftcard product works as expected (product image, add to card text, add to cart URL etc...) with WooCommerce product-related blocks
- [ ] Products (Beta)
- [ ] All Products
- [ ] Featured Product
- [ ] Hand-picked products
- [ ] Best Selling Products
- [ ] Products by Category
- [ ] Product Collection (Beta)
- [ ] Newest Products
- [ ] On Sale Products
- [ ] Products by Tag
- [ ] Top Rated Products
- [ ] On Sale Products
- [ ] Products by Tag
- [ ] Top Rated Products
- [ ] Products by Attribute
- [ ] Single Product

### Changelog entry
> Add - Support for WooCommerce Product Blocks.
